methods.tex

```diff
@@ -294,7 +294,7 @@ \subsubsection{Preferences}
 facilities to adjust their set of preferences and for managers of consumer
 facilities to affect the consumer's set of preferences. Accordingly, the last
 phase is termed the \textit{Preference Adjustment} (PA) phase. By allowing
-facility managers, i.e., a facility's institution and region, to also adjust
+facility managers, i.e., a facility's institution and region, to also adjust  %% the RIF hierarchy has not been introduced yet - not sure where to do so
```

Comment:
I think RIF was covered in the fundamentals paper... I'll try to massage this language a bit so it doesn't directly require an expectation of knowledge of RIF